src/components/Navbar/index.js

```diff
@@ -19,6 +19,10 @@ const Navbar = () => {
                 <S.NavbarLink to="/stats">Stats</S.NavbarLink>
             </S.ListItem>
 
+            <S.ListItem>
+                <S.NavbarLink to="/signup">Sign Up</S.NavbarLink>
+            </S.ListItem>
```

Comment:
i think this should be added to the homepage and the login page but not the main navigation. if we do decide to keep it here it should disappear when we have an `authSession`